store/claim.js

```diff
@@ -4,7 +4,7 @@ import invalidatePreviousRequests from '../utils/invalidatePreviousRequests'
 
 import apiRequest from 'api'
 
-class DataProvider {
+class Claim {
```

Comment:
~_Claim_ is a verb. Either _Claiming_ or _ClaimRequest_.~ However, neither is correct.

It's not claiming (requesting) access to the data provider entity in CORE. It's creating the data provider entity in CORE.

Reply:
we can call it NewDataProvider if you wish

Reply:
NewDataProvider is better but still doesn't feel right to me. If you prefer, go ahead with this name. I recommend focusing on the overall application data model rather than trying to rename a single thing.

Reply:
Technically, you have a works list, a data providers list which can be appended, or update. Each one contains items of the own time, which can be updated. We can start from this.

Reply:
I will keep it as is because I have already two PRs depend on this. Thanks